packages/react/src/themes/teams/components/Attachment/attachmentStyles.ts

```diff
@@ -75,7 +75,7 @@ const attachmentStyles: ComponentSlotStylesPrepared<AttachmentProps, AttachmentV
       ...getIconFillOrOutlineStyles({ outline: true }),
 
       ':hover': {
-        iconFilledStyles,
+        ...iconFilledStyles,
```

Comment:
Oops, was accidentally broken in #2211. FYI @codepretty

Reply:
is this also broken in 0.43? do we want to backport the fix?

Reply:
Good catch, moved it to #2302.